distbench_engine.cc

```diff
@@ -500,6 +500,14 @@ absl::Status DistBenchEngine::InitializeTables() {
   return absl::OkStatus();
 }
 
+// Reminder: traffic_config is passed in, traffic_config_ is a member variable
+// of the DistBenchEngine class
+absl::Status DistBenchEngine::Parse(
```

Comment:
see if you can re-order the initialization so that the Initialize function calls this function, so that it stays up-to-date.

In particular I wonder if the InitializeTables function dependson some of the other fields being initialized.